test/crawler_web/controllers/keyword_controller_test.exs

```diff
@@ -0,0 +1,82 @@
+defmodule CrawlerWeb.KeywordControllerTest do
+  use CrawlerWeb.ConnCase, async: true
+
+  alias Crawler.Keyword.Keywords
+  alias Crawler.Keyword.Schemas.Keyword
+
+  describe "POST /keyword" do
+    test "given valid keyword csv file, creates keyword successfully", %{conn: conn} do
+      user = insert(:user)
+      uploaded_file = keyword_file_fixture("valid.csv")
+
+      conn =
+        conn
+        |> log_in_user(user)
+        |> post(Routes.keyword_path(conn, :create), %{keyword_csv_file: %{file: uploaded_file}})
+
+      assert get_flash(conn, :info) == "3 #{gettext("keywords were uploaded!")}"
+
+      keywords = Keywords.list_keywords(user.id)
+
+      assert length(keywords) == 3
+
+      assert [%Keyword{name: "one"}, %Keyword{name: "two"}, %Keyword{name: "three"}] = keywords
```

Comment:
This doesn't fix the issue I mentioned early 

```suggestion
      assert ([%Keyword{name: "one"}] in  keywords) == true
      assert ([%Keyword{name: "two"}] in  keywords) == true
      assert ([%Keyword{name: "three"}] in  keywords) == true
```

Reply:
Updated in [d493a8c](https://github.com/vnntsu/elixir-ic/pull/36/commits/d493a8cb0958563e30008880bc6ebf7531acadb7)